targets/TARGET_Realtek/TARGET_AMEBA/TARGET_SDT8195B/PinNames.h

```diff
@@ -0,0 +1,281 @@
+    PD_7  = (PORT_D<<4|7),
+    PD_8  = (PORT_D<<4|8),
+    PD_9  = (PORT_D<<4|9),
+  
+    PE_0  = (PORT_E<<4|0),
+    PE_1  = (PORT_E<<4|1),
+    PE_2  = (PORT_E<<4|2),
+    PE_3  = (PORT_E<<4|3),
+    PE_4  = (PORT_E<<4|4),
+    PE_5  = (PORT_E<<4|5),
+    PE_6  = (PORT_E<<4|6),
+    PE_7  = (PORT_E<<4|7),
+    PE_8  = (PORT_E<<4|8),
+    PE_9  = (PORT_E<<4|9),
+    PE_A  = (PORT_E<<4|10),
+  
+    PF_0  = (PORT_F<<4|0),
+    PF_1  = (PORT_F<<4|1),
+    PF_2  = (PORT_F<<4|2),
+    PF_3  = (PORT_F<<4|3),
+    PF_4  = (PORT_F<<4|4),
+    PF_5  = (PORT_F<<4|5),
+    /* unavailable pins */
+//    PF_6  = (PORT_F<<4|6),
+//    PF_7  = (PORT_F<<4|7),
+
+    PG_0  = (PORT_G<<4|0),
+    PG_1  = (PORT_G<<4|1),
+    PG_2  = (PORT_G<<4|2),
+    PG_3  = (PORT_G<<4|3),
+    PG_4  = (PORT_G<<4|4),
+    PG_5  = (PORT_G<<4|5),
+    PG_6  = (PORT_G<<4|6),
+    PG_7  = (PORT_G<<4|7),
+
+    PH_0  = (PORT_H<<4|0),
+    PH_1  = (PORT_H<<4|1),
+    PH_2  = (PORT_H<<4|2),
+    PH_3  = (PORT_H<<4|3),
+    PH_4  = (PORT_H<<4|4),
+    PH_5  = (PORT_H<<4|5),
+    PH_6  = (PORT_H<<4|6),
+    PH_7  = (PORT_H<<4|7),
+
+    PI_0  = (PORT_I<<4|0),
+    PI_1  = (PORT_I<<4|1),
+    PI_2  = (PORT_I<<4|2),
+    PI_3  = (PORT_I<<4|3),
+    PI_4  = (PORT_I<<4|4),
+    PI_5  = (PORT_I<<4|5),
+    PI_6  = (PORT_I<<4|6),
+    PI_7  = (PORT_I<<4|7),
+
+    PJ_0  = (PORT_J<<4|0),
+    PJ_1  = (PORT_J<<4|1),
+    PJ_2  = (PORT_J<<4|2),
+    PJ_3  = (PORT_J<<4|3),
+    PJ_4  = (PORT_J<<4|4),
+    PJ_5  = (PORT_J<<4|5),
+    PJ_6  = (PORT_J<<4|6),
+    /* unavailable pins */
+//    PJ_7  = (PORT_J<<4|7),
+
+    PK_0  = (PORT_K<<4|0),
+    PK_1  = (PORT_K<<4|1),
+    PK_2  = (PORT_K<<4|2),
+    PK_3  = (PORT_K<<4|3),
+    PK_4  = (PORT_K<<4|4),
+    PK_5  = (PORT_K<<4|5),
+    PK_6  = (PORT_K<<4|6),
+    /* unavailable pins */
+    //    PK_7  = (PORT_K<<4|7),
+
+    AD_1  = (PORT_V<<4|1),
+    AD_2  = (PORT_V<<4|2),
+    AD_3  = (PORT_V<<4|3),
+
+    DA_0  = (PORT_U<<4|0),
+    DA_1  = (PORT_U<<4|1),
+
+    // Analog
+    A0 = AD_1,
+    A1 = AD_2,
+    A2 = DA_0,
+    A3 = NOT_CONNECTED,
+
+    // General Pin Input Output (GPIO)
+    GPIO0 = PD_6,
+    GPIO1 = PD_7,
+    GPIO2 = PB_4,
+    GPIO3 = PB_5,
+    GPIO4 = PE_5,
+    //GPIO5 = NFC_IP,
+    //GPIO6 = NFC_IN,
+
+    // LEDs
+    LED0 = GPIO0,
+    LED1 = GPIO1,
+    LED2 = GPIO2,
+   
+    LED_RED   = LED0,
+    LED_GREEN = LED1,
+    LED_BLUE  = LED2,
+
+    // USB bridge and SWD UART connected UART pins
+    USBTX = PB_0,
+    USBRX = PB_1,
+
+    // UART pins
+    UART0_RX  = PA_0,
+    UART0_TX  = PA_4,
+    UART0_CTS = PA_1,
+    UART0_RTS = PA_2,
+
+    UART1_RX  = USBRX,
+    UART1_TX  = USBTX,
+    UART1_CTS = NOT_CONNECTED,
+    UART1_RTS = NOT_CONNECTED,
+
+    UART2_RX  = PA_6,
+    UART2_TX  = PA_7,
+    UART2_CTS = PA_5,
+    UART2_RTS = PA_3,
+
+    // I2C pins
+    I2C0_SCL = PD_5,
+    I2C0_SDA = PD_4,
+
+    I2C1_SCL = PB_2,
+    I2C1_SDA = PB_3,
+
+    I2C2_SCL = NOT_CONNECTED,
+    I2C2_SDA = NOT_CONNECTED,
```

Comment:
Same comment for default peripheral, but looks like only I2C needs to have one in this case.